### ISSUE TITLE
Uncaught TypeError: Cannot read property 'setAttribute' of undefined

### ISSUE THREAD

**Issue by DimanVorosh** [nativescript-vue/nativescript-vue#726]
CONSOLE LOG: TypeError: Class constructor ContainerView cannot be invoked without 'new'
***** Fatal JavaScript exception - application has been terminated. *****
NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'setAttribute' of undefined
at
setStyleScope(file: node_modules/nativescript-vue/dist/index.js:8681:7)
at setScope(file: node_modules/nativescript-vue/dist/index.js:5430:18)
at createElm(file: node_modules/nativescript-vue/dist/index.js:5291:6)
at patch(file: node_modules/nativescript-vue/dist/index.js:5818:6)
at Vue._update(file: node_modules/nativescript-vue/dist/index.js:4830:18)
at updateComponent(file: node_modules/nativescript-vue/dist/index.js:4951:9)
at get(file: node_modules/nativescript-vue/dist/index.js:2336:24)
at Watcher(file: node_modules/nativescript-vue/dist/index.js:2325:11)
at mountComponent(file: node_modules/nativescript-vue/dist/index.js:4958:2)
at push.../node_modules/nativescript-vue/dist/index.js.Vue.$mount(file: node_modules/nativescript-vue/dist/index.js:12519:9)
at init(file: node_modules/nativescript-vue/dist/index.js:4005:12)
at createComponent(file: node_modules/nativescript-vue/dist/index.js:5319:9)
at createElm(file: node_modules/nativescript-vue/dist/index.js:5266:8)
at createChildren(file: node_modules/nativescript-vue/dist/index.js:5394:8)
at createElm(file: node_modules/nativescript-vue/dist/index.js:5295:8)
at patch(file: node_modules/nativescript-vue/dist/index.js:5818:6)
at Vue._update(file: node_modules/nativescript-vue/dist/index.js:4830:18)
at updateComponent(file: node_modules/nativescript-vue/dist/index.js:4951:9)
at get(file:///a<…>
(CoreFoundation) *** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'setAttribute' of undefined
at
setStyleScope(file: node_modules/nativescript-vue/dist/index.js:8681:7)
at setScope(file: node_modules/nativescript-vue/dist/index.js:5430:18)
at createElm(file: node_modules/nativescript-vue/dist/index.js:5291:6)
at patch(file: node_modules/nativescript-vue/dist/index.js:5818:6)
at Vue._update(file: node_modules/nativescript-vue/dist/index.js:4830:18)
at updateComponent(file: node_modules/nativescript-vue/dist/index.js:4951:9)
at get(file: node_modules/nativescript-vue/dist/index.js:2336:24)
at Watcher(file: node_modules/nativescript-vue/dist/index.js:2325:11)
at mountComponent(file: node_modules/nativescript-vue/dist/index.js:4958:2)
at push.../node_modules/nativescript-vue/dist/index.js.Vue.$mount(file: node_modules/nativescript-vue/dist/index.js:12519:9)
at init(file: node_modules/nativescript-vue/dist/index.js:4005:12)
at createComponent(file: node_modules/nativescript-vue/dist/index.js:5319:9)
at createElm(file: node_modules/nativescript-vue/dist/index.js:5266:8)
at createChildren(file: node_modules/nativescript-vue/dist/index.js:5394:8)
at createElm(file: node_modules/nativescript-vue/dist/index.js:5295:8)
at patch(file: node_modules/nativescript-vue/dist/index.js:5818:6)
at Vue._update(file: node_modules/nativescript-vue/dist/index.js:4830:18)
at updateComponent(fi<…>

**Comment by keithgulbro** [comment]
@DimanVorosh ,

Which version of Nativescript-Vue are you using? If you're not using NS 7 ensure your Nativescript-Vue version is below `2.8.1`.

**Comment by ivarsmednis** [comment]
I have same problem after upgrading to Nativescript 7. Upgrading nativescript-vue to 2.8.1 didn't help

**Comment by kelf47** [comment]
same problem here after updating Nativescript 7.

**Comment by donmb1** [comment]
+1

**Comment by donmb1** [comment]
@DimanVorosh did you find a solution in the meantime?

**Comment by rigor789** [comment]
It would be very helpful if you could provide a repro + filled out the issue template so we can look into the issue and a possible fix. Otherwise there's not much we can do with issues like these.

**Comment by ArthurMJ96** [comment]
@DimanVorosh did you find a solution in the meantime?

**Comment by TommyVetZoo** [comment]
Any news on this?

**Comment by rigor789** [comment]
> 
> 
> Any news on this?

Yes: https://github.com/nativescript-vue/nativescript-vue/issues/726#issuecomment-723047385

**Comment by dukeofharen** [comment]
I also ran into this issue. At first, I only ran `ns migrate` (for migrating NS from 6 to 7). After reading the page https://docs.nativescript.org/releases/upgrade-instructions, I also executed `tns update` and after that everything seems to work fine again.

I compared the packages between my old / not working migrated app and a newly bootstrap Nativescript Vue app. These are the differences at first glance in the `package.json` file.
- Added "@nativescript/core"
- Updated "vuex"
- Updated "@nativescript/theme"
- Updated "nativescript-datetimepicker"
- Updated "nativescript-directions"
- Updated "nativescript-ui-sidedrawer"
- Updated "nativescript-vue"
- Updated "babel-loader"
- Updated "nativescript-vue-template-compiler"
- Updated "node-sass"
- Added "@nativescript/android"
- Updated "vue-loader"
- Added "nativescript-worker-loader"
- Added "@nativescript/webpack"

**Comment by razliberman-bv** [comment]
+1

**Comment by razliberman-bv** [comment]
In my case the issue was related to @teammaestro/nativescript-svg package. I've replaced it with @nativescript-community/ui-svg/vue

**Comment by dancosta-fed** [comment]
Any news on this?
I've got NativeScript 8.2.3 version, and Nativescript-Vue at 2.9.0. And still facing the same issue.
